components/NavigationDrawer/src/MDCBottomDrawerViewController.m

```diff
@@ -205,4 +205,13 @@ - (void)contentDrawerTopInset:(CGFloat)transitionToTop {
   [self.delegate bottomDrawerControllerDidChangeTopInset:self topInset:topInset];
 }
 
+- (void)expandToFullScreenWithCompletion:(void (^)(BOOL))completion {
+  if ([self.presentationController isKindOfClass:[MDCBottomDrawerPresentationController class]]) {
+    MDCBottomDrawerPresentationController *bottomDrawerPresentationController =
+        (MDCBottomDrawerPresentationController *)self.presentationController;
+    [bottomDrawerPresentationController expandToFullScreen];
+  }
+  completion(YES);
```

Comment:
I may be wrong about this but wouldn't we want the completion block to fire after the animation is finished. I'm not sure if that would be possible with this implementation but something to consider.

Reply:
Yeah, I considered that, I don't know really where the completion block should go, if it's after the animation then it animates and "jumps" to the completed state, if it's as it stands it "jumps" to the completion as it animates, I think we want to have them animate together.